man/man8/zpool-events.8

```diff
@@ -136,7 +136,7 @@ Issued when the ashift alignment requirement has increased.
 .It Sy vdev.remove
 Issued when a vdev is detached from a mirror (or a spare detached from a
 vdev where it have been used to replace a failed drive - only works if
-the original drive have been readded).
+the original drive have been read).
```

Comment:
No, readded (or re-added) was correct.

Reply:
Indeed! Updated to "re-added". Tks.

Reply:
[nit] PR title: it's "spelling", not "speling"
:)

Reply:
"Speling" is an long standing trope or joke - goes back to at least 1908: https://publicdomainreview.org/collection/the-pioneer-ov-simplified-speling-vol-1-no-1-1912  ;-)

Reply:
I see, apologies, a non-native here.